readthedocsext/theme/templates/projects/partials/header.html

```diff
@@ -193,35 +193,59 @@
       </div>
     </div>
 
-    <div class="ui mini modal" data-modal-id="project-debug">
-      <div class="header">{% trans "Debug information" %}</div>
-      {% comment %}
+    {% if user.is_staff %}
+      <div class="ui mini modal" data-modal-id="project-debug">
+        <div class="header">{% trans "Debug information" %}</div>
+        {% comment %}
         Show spam information on community instance.
         Allows us to quickly understand what's the spam score,
         and also know what rules are matching this project.
-      {% endcomment %}
-      <div class="content">
-        {% if not USE_ORGANIZATIONS %}
-          <h3>{% trans "Spam" %}</h3>
-          <div class="ui sub header">{% trans "Spam score" %}</div>
-          <span>{{ project|get_spam_score }}</span>
-          {% if project.spam_rules.exists %}
-            <div class="ui sub header">{% trans "Spam matching rules" %}</div>
+        {% endcomment %}
+        <div class="content">
+          <h3>{% trans "Impersonate" %}</h3>
+          <div class="ui sub header">{% trans "Project maintainer" %}</div>
+          <div class="ui vertical list">
+            {% for user in project|admins %}
+              <div class="item">
+                <a href="{{ ADMIN_URL|cut:"/admin" }}/impersonate/{{ user.pk }}/" target="_blank">{{ user.username }}</a>
+              </div>
```

Comment:
No need to nest these elements, links can be items too:

```suggestion
              <a class="item" href="{{ ADMIN_URL|cut:"/admin" }}/impersonate/{{ user.pk }}/" target="_blank">{{ user.username }}</a>
```